README.md

```diff
@@ -162,6 +162,18 @@ If you override the default base font, you might also want to turn off font prel
   disableFontPreload = true
 ```
 
+Inlining all styles and scripts directly within the HTML can simplify deployment and speed up initial page load:
```

Comment:
```suggestion
Inlining all styles and scripts directly within the HTML can simplify deployment and speed up initial page load. The benefit here is that you are deploying a single HTML file with all JS/CSS baked into it. The trade-off is that JS/CSS are not individual files that are cacheable. By default this project does not use inline JS/CSS but it can be enabled by setting the following values:
```

@mjethani Thoughts on updating the docs like this?

Reply:
@GrantBirki make sense, done.